010-quarkus-opentracing-reactive-grpc/src/test/java/io/quarkus/qe/GrpcPingPongResourceTest.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.qe;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.Disabled;
+
+import io.quarkus.qe.pong.GrpcPongService;
+import io.quarkus.test.junit.QuarkusTest;
+
+/**
+ * Disabled because OpenTracing does not work with REST -> gRPC.
+ */
+@Disabled
```

Comment:
This test is not working: the trace headers are not transmitted to gRPC endpoint. I can see some bits that could be done to enable this integration (https://github.com/grpc-ecosystem/grpc-opentracing/tree/master/java).

However, taking into account that OpenTracing is deprecated. I think we should remove the gRPC test from here. What do you think? @rsvoboda @mjurc

Reply:
`@Disabled` may optionally be declared with a reason to document why the annotated test class or test method is disabled.

Reply:
I would keep the test in place, we can recheck it with OpenTelemetry

And please open issue for this in https://github.com/quarkusio/quarkus/issues so this gets discussed with developers

Reply:
Raised https://github.com/quarkusio/quarkus/issues/13224 and updated the test to link this issue.